docs/reference/accelerometer.md

```diff
@@ -21,6 +11,20 @@ The parent node of an [Accelerometer](#accelerometer) node should have a [Physic
 
 ### Field Summary
 
+```
+Accelerometer {
+  SFString name       "accelerometer" # used by wb_robot_get_device()
+  MFVec3f lookupTable [ ]             # lookup table
+  SFBool  xAxis       TRUE            # {TRUE, FALSE}
+  SFBool  yAxis       TRUE            # {TRUE, FALSE}
+  SFBool  zAxis       TRUE            # {TRUE, FALSE}
+  SFFloat resolution  -1              # [0, inf)
+}
+```
+**Note**: The above description lists only the fields specific to the Accelerometer node. The complete field list can be found in the [Accelerometer.wrl](https://github.com/cyberbotics/webots/blob/master/resources/nodes/Accelerometer.wrl) definition file.
```

Comment:
Sorry, I missed this one in my first review. This is a better way to link to the correct version of the source code:
```suggestion
**Note**: The above description lists only the fields specific to the Accelerometer node. The complete field list can be found in the [Accelerometer.wrl]({{ url.github_tree }}/resources/nodes/Accelerometer.wrl) definition file.
```

Reply:
Nice, committing it